devicetypes/smartthings/zwave-water-valve.src/zwave-water-valve.groovy

```diff
@@ -54,9 +56,18 @@ metadata {
 }
 
 def updated() {
+	// Device-Watch simply pings if no device events received for 32min(checkInterval)
+	unschedule()
+	runEvery15Minutes("healthPoll")
```

Comment:
Let's use the Z-Wave reporting interval cmd class:
`zwave.wakeUpV2.wakeUpIntervalSet(...)`